iceoryx2-ffi/python/README.md

```diff
@@ -1,56 +1,47 @@
 # iceoryx2-ffi-python
 
-## Running Examples
+Quick reference for commands relevant for development of the `iceoryx2` Python bindings.
+
+## Install Poetry
 
 ```sh
-export PYTHONPATH="$(git rev-parse --show-toplevel)/iceoryx2-ffi/python/python-src"
-python examples/python/event/listener.py
+curl -sSL https://install.python-poetry.org | python3 -
+poetry self add poetry-plugin-shell
 ```
 
-## Setup Development Environment
+## Setup Virtual Environment
 
 ```sh
-# install maturin, see
-# https://github.com/PyO3/maturin
-
 cd $(git rev-parse --show-toplevel)
 
-# create python development environment
-python -m venv .env
-
-# enter environment
-source .env/bin/activate # or source .env/bin/activate.fish
+# Install dependencies and create virtual environment
+poetry --project iceoryx2-ffi/python install
 
-# install dependencies
-pip install pytest
-pip install prospector[with_mypy]
-pip install black
-pip install isort
-pip install bandit
+# (OPTIONAL) Enter the virtual environment - skip the 'poetry run' prefix for all commands
+poetry --project iceoryx2-ffi/python shell
 ```
 
 ## Development
 
 ```sh
-# compile PyO3 bindings
-cd iceoryx2-ffi/python
-maturin develop
+cd $(git rev-parse --show-toplevel)
 
-export PYTHONPATH="$(git rev-parse --show-toplevel)/iceoryx2-ffi/python/python-src"
-# test python bindings
-pytest tests/*
+# Compile PyO3 bindings
+poetry --project iceoryx2-ffi/python build-into-venv
 
-cd $(git rev-parse --show-toplevel)
+# Test python bindings
+poetry --project iceoryx2-ffi/python test
 
-# static code analysis
-prospector -m -D -T --with-tool mypy -s veryhigh $(pwd)/examples/python
-prospector -m -D -T --with-tool mypy -s veryhigh $(pwd)/iceoryx2-ffi/python/tests
+# Run static code analysis
+poetry --project iceoryx2-ffi/python check-linting
```

Comment:
Could you please also add the commands to automatically fix the formatting and imports.